openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -13,12 +13,12 @@ <h2 class="title">
     </h2>
 
     <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__student__response">
-        <div class="ui-staff ui-toggle-visibility is--collapsed">
-            <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
-                <i class="icon fa fa-caret-right" aria-hidden="true"></i>
-                <span>{% trans "Learner's Response" %}</span>
+        <div class="ui-staff ui-slidable" role="button" aria-pressed="false" tabindex="0">
```

Comment:
@cptvitamin for these collapsible sections, I left them as divs because changing to a button caused too many styling issues. However, I made them focusable, added the role of button, and am toggling "aria-pressed".